image_client.py

```diff
@@ -156,10 +156,10 @@ def check_image_db_if_filename_imported(self, collection, filename, exact=False)
 
         return self.decode_response(params)
 
-    def update_exif_image_data(self, exif_ring, collection, filename):
+    def update_4_image_data(self, exif_dict, collection, filename):
         data = {'filename': filename,
```

Comment:
write_image_metadata

Reply:
That method name must have been a typo, probably mean it to be update_image_data, and hit the 4 key somewhere